### ISSUE TITLE
[samsungtv] Connection errors log channel

### ISSUE THREAD

**Issue by jsetton** [openhab/openhab2-addons#5794]
## Expected Behavior
Connection errors, at least for legacy devices, should be logged in debug channel and handled gracefully by setting the thing status to offline. On a side note, other errors should probably be logged in warn or error channels.

## Current Behavior
Connection errors are logged in info channel and thing status remains in unknown status.

```
2019-07-03 12:13:01.538 [INFO ] [rnal.service.RemoteControllerService] - Using Legacy interface
2019-07-03 12:13:02.040 [INFO ] [tv.internal.handler.SamsungTvHandler] - Error was reported: Cannot connect to remote control service
org.openhab.binding.samsungtv.internal.protocol.RemoteControllerException: Connection failed
        at org.openhab.binding.samsungtv.internal.protocol.RemoteControllerLegacy.openConnection(RemoteControllerLegacy.java:98) ~[255:org.openhab.binding.samsungtv:2.5.0.201907030420]
        at org.openhab.binding.samsungtv.internal.service.RemoteControllerService.start(RemoteControllerService.java:228) [255:org.openhab.binding.samsungtv:2.5.0.201907030420]
        at org.openhab.binding.samsungtv.internal.handler.SamsungTvHandler.startService(SamsungTvHandler.java:289) [255:org.openhab.binding.samsungtv:2.5.0.201907030420]
        at org.openhab.binding.samsungtv.internal.handler.SamsungTvHandler.checkCreateManualConnection(SamsungTvHandler.java:275) [255:org.openhab.binding.samsungtv:2.5.0.201907030420]
        at org.openhab.binding.samsungtv.internal.handler.SamsungTvHandler.checkAndCreateServices(SamsungTvHandler.java:218) [255:org.openhab.binding.samsungtv:2.5.0.201907030420]
        at org.openhab.binding.samsungtv.internal.handler.SamsungTvHandler.initialize(SamsungTvHandler.java:152) [255:org.openhab.binding.samsungtv:2.5.0.201907030420]
        at sun.reflect.GeneratedMethodAccessor82.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [136:org.openhab.core:2.5.0.201907030301]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [136:org.openhab.core:2.5.0.201907030301]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: java.net.SocketTimeoutException: connect timed out
        at java.net.PlainSocketImpl.socketConnect(Native Method) ~[?:?]
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[?:?]
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[?:?]
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[?:?]
        at java.net.Socket.connect(Socket.java:589) ~[?:?]
        at org.openhab.binding.samsungtv.internal.protocol.RemoteControllerLegacy.openConnection(RemoteControllerLegacy.java:95) ~[?:?]
        ... 14 more
```

## Context

Some of the older Samsung TV models aren't accessible over the network when turned off. So the binding should account for such behavior. This used to work that way before the major update that was pushed a few months back, adding websocket support for the more recent models.

## Your Environment
Docker 2.5.0-SNAPSHOT Build 1629

**Comment by arjanmels** [comment]
I have created pull request #5813 which should solve this problem. Compiled version is at:  https://drive.google.com/open?id=1BAg5Xs1UtuZi61p5FpXjVJGWV31-HLK8

**Comment by jsetton** [comment]
Thanks for the quick turnaround.

**Comment by cweitkamp** [comment]
Closed by #5813 .
